src/js/components/Tabs/doc.js

```diff
@@ -48,6 +48,21 @@ currently active tab changes.`,
 };
 
 export const themeDoc = {
+  'global.borderSize': {
+    description: 'The size of the border',
+    type: 'object',
+    defaultValue: undefined,
+  },
+  'global.colors.border': {
+    description: 'color of the border',
+    type: 'string',
+    defaultValue: undefined,
```

Comment:
for `global.colors` we usually do have defaultValues.
in this case it should be 
```
defaultValue: {
      dark: 'rgba(255, 255, 255, 0.33)',
      light: 'rgba(0, 0, 0, 0.33)',
    },
```
you can find it on `base.js`